HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -269,6 +278,7 @@ def customizeHLTforCMSSW(process, menuType="GRun"):
     # process = customiseFor12718(process)
 
     process = checkHLTfor43774(process)
+    process = customizeHLTfor44222(process)
```

Comment:
Also here?

Reply:
Not sure by "Also here?" but maybe done in a7ab886c1a50d2c052892daf3fd873fd62bfc46a

Reply:
Can you please squash the last two commits? @stahlleiton . Then we should be good to go.

Reply:
done, the last two commits are squashed in 169cad26493e9982a492a09ae50d3fcfcbc47a8a

Reply:
thanks a lot!